docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.9"
+
+services:
+  text-classification:
+    image: oblivion208/asr:2.3.0-gpu
```

Comment:
@fengredrum Can you change the name of service to like `TensorFlowASR` and use context build point to the `Dockerfile` instead of your own image, so that everyone have a flexibility in using both `Dockerfile` and `docker-compose` to build their custom environment? You can comment with your name at the top of this file like I did in the `config.yml` files in the examples:
```yaml
# Copyright 2020 Huy Le Nguyen (@usimarit)
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
@usimarit Sorry about that, I'll fix it😄